app/cdap/components/RecipeManagement/RecipeDetails/index.tsx

```diff
@@ -0,0 +1,51 @@
+/*
+ *  Copyright © 2022 Cask Data, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ *  use this file except in compliance with the License. You may obtain a copy of
+ *  the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *  License for the specific language governing permissions and limitations under
+ *  the License.
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import { Container, Drawer } from '@material-ui/core';
+import DrawerHeader from 'components/RecipeManagement/RecipeDetails/DrawerHeader';
+import DetailContainer from 'components/RecipeManagement/RecipeDetails/DetailContainer';
+import { IRecipe } from 'components/RecipeList/types';
+
+interface IRecipeDetailsProps {
```

Comment:
reuse types from recipelist whever possible if we can't reuse put them in recipe management types.ts file and also we should avoid redundant code if it is already defined anywhere